README.md

```diff
@@ -2,13 +2,13 @@
 id: instructure-ui
 ---
 
-## instructure-ui
+## Instructure UI
 
-[![build-status][build-status]][build-status-url]&nbsp;&nbsp;&nbsp;[![MIT License][license-badge]][license]&nbsp;&nbsp;&nbsp;[![Code of Conduct][coc-badge]][coc]
+[![MIT License][license-badge]][license]&nbsp;&nbsp;&nbsp;[![Code of Conduct][coc-badge]][coc]
 
-## Usage
+Instructure UI is an open source UI framework and design system for React. Its components are used by millions of learners every day when they use Instructure products like Canvas LMS.
```

Comment:
learners -> students? sounds a bit better to me

Reply:
This is a copypaste from the current main page of instructure.design

Reply:
Oh ok, sry, not my first language :D